lib/checkIssueAssignee.js

```diff
@@ -78,10 +82,11 @@ const checkAssignees = async (context) => {
       await context.github.issues.removeAssignees(assigneeParam);
     }
   } catch (error) {
+    // eslint-disable-next-line no-console
```

Comment:
this is done to disable the `no-console` issue. 
Should we update in .eslintrc directly?

Reply:
@anandwana001 Could you disable this in `.eslintrc` and remove all this disables?

Reply:
ok

Reply:
Done